src/main/java/com/ibm/crypto/plus/provider/XDHPrivateKeyImpl.java

```diff
@@ -94,8 +96,8 @@ public XDHPrivateKeyImpl(OpenJCEPlusProvider provider, byte[] encoded)
         try {
             byte[] alteredEncoded = processEncodedPrivateKey(encoded); // Sets params, key, and algid, and alters encoded
             // to fit with GSKit and sets params
-            int encodingSize = CurveUtil.getDEREncodingSize(curve);
-            this.xecKey = XECKey.createPrivateKey(provider.getOCKContext(), alteredEncoded, encodingSize);
+            int curveSize = CurveUtil.getCurveSize(curve);
+            this.xecKey = XECKey.createPrivateKey(provider.getOCKContext(), alteredEncoded, curveSize);
```

Comment:
A question, this PR is for implement the `calculatePublicKey()` method, but why here, change from the `getDEREncodingSize(curve)` to `getCurveSize(curve)`?